helm/cloud-provider-cloud-director-app/charts/cloud-director-named-disk-csi-driver/templates/configmap.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vcloud-csi-configmap
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+data:
+  vcloud-csi-config.yaml: |+
+    vcd:
+      host: {{ .Values.configmap.site }}
+      org: {{ .Values.configmap.org }}
+      vdc: {{ .Values.configmap.ovdc }}
+      vAppName: {{ .Values.configmap.vAppName }}
+    clusterid: {{ .Values.configmap.clusterid }}
+immutable: true
```

Comment:
Is this required?

Reply:
It's in the upstream manifests, I'll parameterize it